Source/Controller/HelpDrawerCoordinator.swift

```diff
@@ -0,0 +1,300 @@
+//
+//  HelpDrawerCoordinator.swift.
+//  Planetary
+//
+//  Created by Matthew Lorentz on 7/14/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+import Analytics
+import Logger
+
+/// A model for help UI that is presented in a sheet or popover.
+enum HelpDrawer {
+    case home, discover, notifications, hashtags, network
+    
+    init?(viewController: HelpDrawerHost) {
```

Comment:
The chain of ifs makes me wonder if this enum should be better set by HelpDrawerHost instances. Like a property for instance.

Reply:
Ah yeah that's a good idea.

Reply:
Done.